src/string/strchr.c

```diff
@@ -0,0 +1,14 @@
+char *
+strchr(const char *s, int c)
+{
+    while (*s)
+    {
+        if (*s == c)
+        {
+            return ((char *) s);
+        }
+        s++;
+    }
+
+    return (0);
```

Comment:
If we want to be 100% standard compliant, we should use the NULL macro, as it is not 100% guarantee that it is (void*)0.

From the c standard: 
> The macros are
> NULL
> which expands to an **implementation-defined** null pointer constant;

Reply:
`return (NULL)` instead of 0